libobs-d3d11/d3d11-subsystem.cpp

```diff
@@ -302,6 +302,10 @@ void gs_device::InitCompiler()
 				module, "D3DDisassemble");
 #endif
 			if (d3dCompile && d3dCreateBlob) {
+				if (ver < 47)
+					blog(LOG_WARNING,
+					     "InitCompiler: Using %s",
+					     d3dcompiler);
```

Comment:
It might be worth it to log this unconditionally rather than only if the version is old, since if there's some version-specific bug in the future, having that information already logged would make it easier to identify. (assuming we don't just give up and bundle the compiler DLL)